documentation/src/main/asciidoc/stories/assembly_hotrod_client_api.adoc

```diff
@@ -13,6 +13,7 @@ include::{topics}/ref_hotrod_versioned_api.adoc[leveloffset=+1]
 include::{topics}/ref_hotrod_streaming_api.adoc[leveloffset=+1]
 include::{topics}/ref_hotrod_listeners.adoc[leveloffset=+1]
 include::{topics}/ref_hotrod_client_transactions.adoc[leveloffset=+1]
+include::{topics}/ref_hotrod_countermanager.adoc[leveloffset=+1]
```

Comment:
move this after "streaming_api"? IMO, it looks better together with the other API sections